BrainPortal/app/assets/stylesheets/cbrain.css.erb

```diff
@@ -2313,21 +2313,32 @@ img {
 /* % Background Activity Index */
 /* % ######################################################### */
 
+/* reset progress bar defaults */
 #bac_form progress {
   -webkit-appearance: none;
   -moz-appearance: none;
   appearance: none;
 }
 
+/* progress bar styles */
 .background_activity_progress {
-  height: 3em;
-  width: 40em;
-  background-color: #eee;
-  border: 0.5em solid black;
+  height: 1.3em;
```

Comment:
I'm ok with adding comments to the stylesheets, but why are you changing the styles themselves? That's not part of the PR's purpose. Was there something wrong with them? I created them to work with Safari, Chrome and Firefox, did you try all three?

Reply:
That's from somebody else commit.